src/redux/actions/projects/deleteProject.js

```diff
@@ -0,0 +1,42 @@
+import {
+  PROJECTS_DELETE, PROJECTS_SET_ACTIVE,
+} from '../../actionTypes/projects';
+
+import {
+  SAMPLES_DELETE,
+} from '../../actionTypes/samples';
+
+const deleteProject = (
+  projectUuid,
+) => async (dispatch, getState) => {
+  // Delete samples
+  const { projects } = getState();
+  const { activeProject } = projects.meta;
+  const { samples } = projects[projectUuid];
+
+  if (samples.length) {
+    samples.forEach((sampleUuid) => {
+      dispatch({
```

Comment:
instead of dispatching many `SAMPLES_DELETE`, what do you think about modifying the reducer so that it expects and array of all the samples it needs to delete?

Reply:
Hmm... users can delete project one by one, or delete projects, where samples will be deleted in bulk. Ideally, it would be good to have 2 actions for delete : single and bulk. However, I don't think it's necessary right now as we're only dealing with a small number of samples. This might be needed though, if users have tens of samples.

It's a simple addition, and it would future proof. However, for this stage, I think it might be over-engineering. What do you think?

Reply:
I don't know if we actually need to split it up into 2 actions.

Looking for similar issues and how they were resolved I ran into the `changeGeneSelection` action.
This one is a single action that always takes an array of gene keys, even when it can be called with only one element.

Reply:
For now, I don't think there is the case for users to delete multiple users at once. However, there is chance for multiple deletes. I'll add this functionality to delete samples

Reply:
Added changes in 53c3169